EventTarget.js

```diff
@@ -1,88 +1,91 @@
-/**
- * @author mrdoob / http://mrdoob.com
- * @author Jesús Leganés Combarro "Piranna" <piranna@gmail.com>
- */
 
+function EventTarget() {}
 
-function EventTarget()
-{
-  var listeners = {};
+Object.assign(EventTarget.prototype, {
 
-  this.addEventListener = function(type, listener)
-  {
-    if(!listener) return
+  addEventListener: function (type, listener) {
+    if (!this._listeners) Object.defineProperty(this, '_listeners', {value: {}})
+    var listeners = this._listeners
```

Comment:
I will NEVER accept methods or attributes that would allow access to implementation specific structures, only the ones defined on the W3C specification. Use private attributes and methods or a WeakMap at the module level instead if you need to store private status.